AncientReligions/events/ACR_other_event.txt

```diff
@@ -213,4 +212,207 @@ character_event = {
 		ai_chance = { factor = 1 }
 		clr_character_flag = gone_pagan
 	}
-}
\ No newline at end of file
+}
+
+# Ask the player if they want to "paganify" the world
+character_event = {
+	id = ancrel.9001
+	desc = ancrel.9001.desc
+	is_triggered_only = yes
+
+	trigger = {
+		NOT = { has_global_flag = acr_paganify }
+	}
+
+	immediate = {
+		set_global_flag = acr_paganify
+	}
+
+	option = {
+		name = ancrel.9001.a
+		any_independent_ruler = {
+			any_realm_character = {
+				if = { # Celtic
+					limit = {
+						OR = {
+							culture_group = celtic
+							culture_group = iberian
+							culture_group = west_germanic
+							culture = occitan
+							culture = frankish
+						}
+					}
+					character_event = { id = ancrel.9002 }
+				}
+				if = { # Hellenic
+					limit = {
+						OR = {
+							culture_group = byzantine
+							culture = italian
+						}
+
+					}
+					character_event = { id = ancrel.9003 }
+				}
+				if = { # Kemetic
+					limit = {
+						OR = {
+							culture = bedouin_arabic
+							culture = levantine_arabic
+							culture = egyptian_arabic
+							culture_group = east_african
+						}
+					}
+					character_event = { id = ancrel.9004 }
+				}
+				if = { # West African
+					limit = {
+						OR = {
+							culture = maghreb_arabic
+							culture = andalusian_arabic
+							culture_group = west_african
+						}
+					}
+					character_event = { id = ancrel.9005 }
+				}
+				if = { # Slavic
+					limit = {
+						OR = {
+							culture_group = west_slavic
+							culture_group = east_slavic
+							culture_group = south_slavic
+						}
+					}
+					character_event = { id = ancrel.9006 }
+				}
+				if = { # Norse
+					limit = {
+						OR = {
+							culture_group = north_germanic
+							culture_group = central_germanic
+						}
+					}
+					character_event = { id = ancrel.9007 }					
+				}
+				if = { 
+					limit = {
+						culture_group = iranian
+					}
+					character_event = { id = ancrel.9008 }
+				}
+			}
+		}
+	}
+}
+
+# Celtic
+character_event = {
+	id = ancrel.9002
+	desc = ancrel.9002.desc
+	is_triggered_only = yes
```

Comment:
Some possible pictures for these events:
GFX_evt_Pagan_dolmen
GFX_evt_roman_temple
GFX_evt_temple_egyptian_pagan
GFX_evt_desert ?
GFX_evt_cathedral_norse_pagan (also for Slavic)
GFX_evt_marriage_zoroastrian_group